### PR TITLE
Feat/payment fraud

### PR DESCRIPTION
**Pull request by makma**

### CODE REVIEW

**Review by ilfa — APPROVED** [review]

File: pages/payment-fraud/index.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+      method: 'POST',
+      body: JSON.stringify(orderData),
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+    });
+
+    const responseJson = await response.json();
+    const responseStatus = await response.status;
```
Comment (ilfa) [inline pages/payment-fraud/index.js:72]: You don't need await here

File: pages/api/payment-fraud/place-order.js [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+    type: Sequelize.STRING,
+  },
+  timestamp: {
+    type: Sequelize.DATE,
+  },
+});
+
+PaymentAttempt.sync({ force: false });
+
+const mockedCard = {
```
Comment (ilfa) [inline pages/api/payment-fraud/place-order.js:44]: Can't find usage of this

Comment (makma) [reply pages/api/payment-fraud/place-order.js:44]: Yep, I decided to remove it eventually, thank you, will remove it.

File: pages/payment-fraud/index.js (hunk 1 above)
Comment (makma) [reply pages/payment-fraud/index.js:72]: Sure, thx, will fix

**Review by ilfa — APPROVED** [review]

**Review by ilfa — APPROVED** [review]

**Pull request merged**